README.md

```diff
@@ -58,14 +58,19 @@ Then you need to enable this component with the id of the ble_client component.
 
 ```yaml
 uni_trend_sound_meter:
-    # [Required] Reference to the ble_client.id set above 
+    # [Required] Name or ID is required.
+    name: "Sound Level"
+    # [Optional] Enhance entity with icon and unit.
+    icon: 'mdi:microphone-variant'
+    unit_of_measurement: 'dBA'
```

Comment:
unit_of_measurement is optional as well, better to leave a comment here, as it's set in `__init__.py`.